nginx/nginx.conf

```
@@ -106,9 +106,10 @@ http {
         location /tuesday { return 302 https://pledge.wnyc.org/support/radiolab; }
 
         # Podcast Membership
-        location ~ ^/donate/? { return 302 https://pledge.wnyc.org/support/radiolab?utm_medium=audio&utm_source=podcast-spot&utm_campaign=rl-cye-addgift; }
+        location ~ ^/donate/? { return 302 https://pledge.wnyc.org/support/radiolab?utm_medium=audio&utm_source=redirect&utm_campaign=radiolab-dot-org-slash-donate ; }
```

Extra space here before the semicolon. Don't know whether that can cause issues in nginx.

```suggestion
        location ~ ^/donate/? { return 302 https://pledge.wnyc.org/support/radiolab?utm_medium=audio&utm_source=redirect&utm_campaign=radiolab-dot-org-slash-donate; }
```